modules/packages/Sort.chpl

```diff
@@ -2390,25 +2502,32 @@ module TwoArrayPartitioning {
     const curDomain = {start_n..end_n};
     const intersect = curDomain[localSubdomain];
     if curDomain == intersect {
-      if n > state.baseCaseSize {
-        msbRadixSort(start_n, end_n,
-                     A.localSlice(curDomain), criterion,
-                     startbit, state.endbit,
-                     settings=new MSBRadixSortSettings());
-      } else {
-        ShellSort.shellSort(A.localSlice(curDomain), criterion, start=start_n, end=end_n);
+      local {
```

Comment:
Do you remember the purpose of this local block? Is this for performance or just an assertion?

I have a slight preference for removing it, since we're trying to move away from local blocks.

Reply:
I do see regressions without the local block:

`./dist-performance -nl 1` 400 MB/s with, 350 MB/s without
`./dist-performance -nl 512` 175 MB/s with, 165 MB/s without

Reply:
(So let's keep the local block for now)